.github/_release.md

```diff
@@ -0,0 +1,29 @@
+ # Frontman release process
```

Comment:
I'd rename the file `RELEASE.md` for consistency with the other files.

Reply:
Done ✅